UnitTests/fixtures/LocStructuredOnly.strings

```diff
@@ -0,0 +1,3 @@
+"settings.navigation-bar.title.even.deeper" = "Settings";
+"SETTINGS.USER_PROFILE_SECTION.HEADER_TITLE" = "User Profile Settings";
+"seTTings.uSer-proFile-sectioN.footer_text" = "Here you can change some user profile settings.";
```

Comment:
Ok this will probably be ok for our first step then, even if it doesn't cover _all_ the cases and generates invalid code if we have leaf and node with the same name, that's still a great start for all those cases where we don't have this special case, so I'm ok for a first version.

Would sill be interested to add a `"settings.user_profile_section.points" = "Welcome %@! You have %d points on your account.";` line in `LocStructuredOnly.strings` and ensure the generated code looks as expected though (as looking at the code it seems it would work and be handled by your template, so let's include that in the Unit Test :wink:)